### ISSUE TITLE
[WIP Limb Stuff] Unable to Wear Exodia Cataphract Armor with Peg Legs

### ISSUE THREAD

**Issue by likowski1** [CleverRaven/Cataclysm-DDA#75337]
### Describe the bug

Despite being Worn on the torso the Cataphract Armor Conflicts with Peg legs

### Attach save file

[World of Lunancy-trimmed.tar.gz](https://github.com/user-attachments/files/16435106/World.of.Lunancy-trimmed.tar.gz)

### Steps to reproduce

1. Start as a Character with missing legs
2. Acquire Cataphract armor
3. Attempt to Wear the Cataphract Armor

### Expected behavior

the Cataphract Armor Shouldn't Conflict with Peg legs

### Versions and configuration

- OS: Windows
    - OS Version: 10.0.22621.3737 (22H2)
- Game Version: 0b942b3 [64-bit]
- Graphics Version: Tiles
- Game Language: System language []
- Mods loaded: [
    Dark Days Ahead [dda],
    Bionic Professions [package_bionic_professions],
    Aftershock [aftershock],
    Translate Complex Dialogue [translate_dialogue],
    Xedra Evolved [xedra_evolved],
    SpeedyDex [speedydex],
    Stats Through Kills [stats_through_kills],
    Tamable Wildlife [Tamable_Wildlife],
    Mind Over Matter [mindovermatter],
    Extra Mutated Scenarios [extra_mut_scens],
    Bombastic Perks [bombastic_perks],
    Portal Storms Ignore NPCs [personal_portal_storms],
    Slowdown Fungal Growth [no_fungal_growth],
    Work in Progress Limb Stuff [limb_wip],
    Alternative Map Key [alt_map_key],
    Magiclysm [magiclysm],
    Magiclysm Revamp and Additions [magicrevamp],
    Disable NPC Needs [no_npc_food],
    Mythical Martial Arts [MMA]
]

**Comment by likowski1** [comment]
After thinking for a moment i may have been a little biased because i had just purchased it with atleast 10 P90 PDW's but on the cataphract armor i Imagined that the Hip Protection was basically a Skirt type of thing like the Lorica Segmenta

**Comment by PGR-14** [comment]
Actually, I think it is. I'm under the impression that it's Greek/Roman style due to it being made of Thessalonian bronze.

**Comment by likowski1** [comment]
> Actually, I think it is. I'm under the impression that it's Greek/Roman style due to it being made of Thessalonian bronze.

agreed, actually i think it may be the peg leg that may be changed because despite being mentioned in the description as "Above the Knee" its Coverage goes all the way to the hips
![image](https://github.com/user-attachments/assets/8ed04079-133c-4caf-b10d-b9e1b0b93348)

**Comment by PGR-14** [comment]
Also, it'd probably make sense to be able to wear hard armor over prosthetics, stuff like greaves & arm guards would make sense on metal & modern prosthetics, 'cause they're about the same shape & proportions of a regular appendage..
